scopecheck/scopecheck.go

```diff
@@ -0,0 +1,94 @@
+package scopecheck
+
+import (
+	"fmt"
+
+	"github.com/taskcluster/taskcluster-cli/extpoints"
+	tcclient "github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/auth"
+	scopeutility "github.com/taskcluster/taskcluster-lib-scopes"
+)
+
+func init() {
+	extpoints.Register("scope-check", scopecheck{})
+}
+
+type scopecheck struct{}
+
+func (scopecheck) ConfigOptions() map[string]extpoints.ConfigOption {
+	return nil
+}
+
+func (scopecheck) Summary() string {
+	return "Shows whether a given scope satifies another."
+}
+
+func (scopecheck) Usage() string {
+	return `Usage:
+  taskcluster scope-check <scope1> satifies <scope2>
+`
+}
+
+func expandScope(Auth *auth.Auth, scope2 string) (string, error) {
+
+	scopeParams := &auth.SetOfScopes{
+		Scopes: scope2,
+	}
+
+	response, errors := Auth.ExpandScopes(scopeParams)
+	if errors != nil {
+		fmt.Printf("Error expanding scopes: %s\n", err)
+		return false
+	}
+
+	_, expandedScopes := response.Scopes
+	return expandedScopes.(string), nil
+}
+
+func (scopecheck) Execute(context extpoints.Context) bool {
+	argv := context.Arguments
+
+	command := argv["SCOPECHECK"].(string)
+	provider := extpoints.CommandProviders()[command]
```

Comment:
Still no clue what this is good for... Why are you touching CommandProviders()?

What is `argv["SCOPECHECK"].(string)`...

Please take some time and make sure you understand the code and why you have it. Because this doesn't seem to make sense.